bin/license/license_auth.sh

```
@@ -0,0 +1,6 @@
+#!/bin/bash
+license_project="$1"
```

Shellcheck (0.10.0)

[warning] license_project appears unused. Verify use (or export if used externally).

(SC2034)